unit3-server-mvc-with-node-express-and-sql/README.md

```diff
@@ -2,50 +2,58 @@
 
 ### Week 1
 
+- Serverside MVC with Express: 3 days
+  - Why MVC for server side development: Separation of concerns
+  - Routing & (simple) regular expressions
+  - Controllers
+  - Views using a templating engine and ViewModels (Mustache? Jade?)
```

Comment:
Do we want to incorporate basic React views instead of using a tempting engine? Would be a little more complex but could be good to start reinforcing some of the React stuff